pkg/apis/build/v1alpha2/image_builds.go

```diff
@@ -34,7 +34,7 @@ type BuildReason string
 func (im *Image) Build(sourceResolver *SourceResolver, builder BuilderResource, latestBuild *Build, reasons, changes string, nextBuildNumber int64) *Build {
 	buildNumber := strconv.Itoa(int(nextBuildNumber))
 
-	return &Build{
+	build := &Build{
```

Comment:
Any reason to not just return this?

Reply:
I pulled it out to a variable to debug something and never put it back. good catch